publish-root.gradle

```diff
@@ -10,6 +10,29 @@ if (localProps.exists()) {
   Properties p = new Properties()
   new FileInputStream(localProps).withCloseable { is -> p.load(is) }
   p.each { name, value -> ext[name] = value }
+} else {
+  Properties p = new Properties()
+  p.putAll(System.getenv())
```

Comment:
read the release properties from env variables - the properties need to match the current naming as they're used by other dependencies